pkg/util/util.go

```diff
@@ -680,7 +680,10 @@ func GetRemoteFilesMarkedForDeletion(delSrcRelPaths []string, remoteFolder strin
 
 // HTTPGetRequest uses url to get file contents
 func HTTPGetRequest(url string) ([]byte, error) {
-	var httpClient = &http.Client{Timeout: HTTPRequestTimeout}
+	var httpClient = &http.Client{Transport: &http.Transport{
+		ResponseHeaderTimeout: HTTPRequestTimeout,
+	},
+		Timeout: HTTPRequestTimeout}
```

Comment:
Given `ResponseHeaderTimeout` is subset of `Timeout`, can we increase `Timeout` instead of let them share the same timeout? Otherwise we might hit the issue where `ResponseHeaderTimeout` is not reached but `Timeout` is reached.

Reply:
Kept response header timeout at 10s, adjusted the other timeout to 20s

Reply:
FWIW, some docs I've read have suggested against using `Timeout`, and only using  fine-grained configs like `ResponseHeaderTimeout` and `IdleTimeout` (or giving `Timeout` a large value), but that probably warrants a separate issue to discuss.